src/components/home/home.css

```diff
@@ -52,3 +52,19 @@
   align-self: center;
   border-radius: 4;
 }
+
+.card{
+  margin: 10;
+  padding-top: 10;
+  width: 100 ;
```

Comment:
it needs to be `.home__container .card ` so we can override the BootStraps css 
marigin and padding-top has to be 10 px
width is 100%